### PR TITLE
feat(billing.history): add tracking

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/activate-billing-revamp`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5458
| License          | BSD 3-Clause

## Description

add tracking for billing revamp

### CODE REVIEW

**Review by ganeshkumar1989 — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html [hunk 1]
```diff
@@ -136,7 +136,7 @@
                 ng-if="$row.url"
                 href="{{ :: $row.url }}"
                 external
-                on-click="$ctrl.trackInvoiceOpening()"
+                on-click="$ctrl.trackInvoiceOpening('html')"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html:139]:
```suggestion
                data-on-click="$ctrl.trackInvoiceOpening('html')"
```

File: packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html [hunk 2]
```diff
@@ -146,7 +146,7 @@
                 ng-if="$row.pdfUrl"
                 href="{{ :: $row.pdfUrl }}"
                 external
-                on-click="$ctrl.trackInvoiceOpening()"
+                on-click="$ctrl.trackInvoiceOpening('pdf')"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html:149]:
```suggestion
                data-on-click="$ctrl.trackInvoiceOpening('pdf')"
```

File: packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html [hunk 3]
```diff
@@ -156,7 +156,7 @@
                 ng-if="$row.pdfBetaUrl"
                 href="{{ :: $row.pdfBetaUrl }}"
                 external
-                on-click="$ctrl.trackInvoiceOpening()"
+                on-click="$ctrl.trackInvoiceOpening('pdf')"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html:159]:
```suggestion
               data-on-click="$ctrl.trackInvoiceOpening('pdf')"
```

**Review by radireddy — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html (hunk 1 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html:139]: As mentioned by @antleblanc https://github.com/ovh/manager/pull/3466#discussion_r472802183 We are shifting on this behavior. Only for webcomponents, meaning that on <oui-action-menu-item>, we are ok to stop using them.

File: packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html [hunk 4]
```diff
@@ -136,7 +136,7 @@
                 ng-if="$row.url"
                 href="{{ :: $row.url }}"
```
Comment (Jisay) [inline packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html:137]: Can be `ng-href` here I think

**Review by Jisay — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html (hunk 4 above)
Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/billing/main/history/billing-main-history.html:137]: It's a `<oui-action-menu-item>` element

**Review by JeremyDec — APPROVED** [review]

**Comment by marie-j** [comment]
Rebased due to rebase on target branch

**Pull request merged**